src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/ObjectWriter/Dwarf/DwarfExpressionBuilder.cs

```diff
@@ -161,6 +161,10 @@ public static int DwarfRegNum(TargetArchitecture architecture, int regNum)
                         _ => regNum - (int)RegNumX86.REGNUM_COUNT + 32 // FP registers
                     };
 
+                case TargetArchitecture.LoongArch64:
+                    // Normal registers are directly mapped
```

Comment:
The JIT mapping for FP regs starts at `F0 == 32`, the DWARF mapping starts at 64 AFAIK. Ideally this should be fixed by adding the code to correct the offset. Not sure about SIMD and if it's exposed at all.

Reply:
I think in LoongArch64, the DWARF mapping starts at 32, because `UNW_LOONGARCH_F0 = 32` and `UNW_LOONGARCH_F31 = 63`. Not sure about SIMD, and it's not exposed yet.

Reply:
> The JIT mapping for FP regs starts at `F0 == 32`, the DWARF mapping starts at 64 AFAIK.

Yes, the LA64's DWARF-document supporting the SIMD, it starts from 64.

> Ideally this should be fixed by adding the code to correct the offset. Not sure about SIMD and if it's exposed at all.

we had planned to update these within the Intrinsic pushing but the LoongArch64's Intrinsic patch is pending for waiting the LA64's SIMD manual.
I think it is close to the time publishing SIMD manual, and we will modify these together.

Thanks for your advices.

Reply:
Yes, you are right.
These had been updated after LA merged into runtime, but the other places we didn't update, in fact you had told us and we planned to do this together with pushing the Intrinsic-LoongArch64 while the Intrinsic is pending temporarily.
We will do these in future.
Thanks for your advice!